plsync/mlabconfig.py

```diff
@@ -8,6 +8,13 @@
 import sys
 import time
 
+ZONE_TTL = 60 * 5
+ZONE_MIN_TTL = 60 * 5
+ZONE_REFRESH = 60 * 5
+ZONE_RETRY = 60 * 60
```

Comment:
You mention above that "retry should be less than retry"... I assume you mean that retry should be less than refresh, which is what I was getting at in my original comment.  I would normally expect refresh to be a multiple of retry...something like refresh at 1 hour and retry at 10 minutes.  That is, if a refresh fails for some reason, then retry the update every 10 minutes thereafter until success.  Having the retry the same as the refresh, or the retry greater than the refresh, doesn't make sense to me.  How about some values like this:

ZONE_REFRESH = 60 * 60
ZONE_RETRY = 60 * 10

In either case I doubt it's a big deal because they likely don't get used.